docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -99,6 +99,25 @@ Group parameters
 
         :return: A :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParameterNumbers`.
 
+    .. method:: public_bytes(encoding, format)
+
+        .. versionadded:: 1.7
+
+        Allows serialization of the key to bytes. Encoding (
+        :attr:`~cryptography.hazmat.primitives.serialization.Encoding.PEM` or
+        :attr:`~cryptography.hazmat.primitives.serialization.Encoding.DER`) and
+        format (
+        :attr:`~cryptography.hazmat.primitives.serialization.PublicFormat.SubjectPublicKeyInfo`)
```

Comment:
The serialization of DH parameters isn't actually a `subjectPublicKeyInfo` structure (which requires an algorithm identifier), but is instead just an ASN.1 sequence of two integers (`p` and `g`). Elsewhere in the code you refer to this as `PKCS3`.

Adding serialization for parameters is a new feature (we don't support it for `DSAParameters` at the moment) and since it is a "new" format I'd like to have this be added in a future PR. Could you remove this for now?